AirCasting/SDSync/SDSyncFileValidationService.swift

```diff
@@ -95,7 +95,7 @@ struct SDSyncFileValidationService: SDSyncFileValidator {
     
     private func lineIsCorrupted(_ line: String) -> Bool {
         let fields = line.split(separator: ",")
-        return !line.isEmpty && fields.count != expectedFieldsCount
+        return !line.isEmpty && fields.count != expectedFieldsCount && fields.count != 1
```

Comment:
We expect a line to have only one value: UUID, so we want to reflect that in stats. I guess in some cases the data may be corrupted in a way that it leaves only one line AND it's not UUID but it wouldn't be super easy to implement and it's such an edge case that I don't mind leaving that as is.